common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -45,6 +45,7 @@ export default function Dashboard() {
             className="Dashboard-mobile-modifiedPanel"
             copyable={true}
             dashboard={true}
+            privacyTriggerEnabled={false}
```

Comment:
Is this necessary with your default value being `false`?

Reply:
nahhhh